packages/extension-commerce-bm-seo/.prettierignore

```diff
@@ -0,0 +1,5 @@
+coverage
+dist
+generator-assets
+docs
+vendor
```

Comment:
Are these all valid entries?

Reply:
This was copied from `extension-starter` and I see that the other extensions have the same entries. However it does seem like these 3 entries don't exist

```
generator-assets
docs
vendor
```